app/javascript/packs/application.js

```diff
@@ -7,6 +7,11 @@
 // To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
 // layout file, like app/views/layouts/application.html.erb
 
+import 'turbolinks-animate'
+document.addEventListener('turbolinks:load', function() {
+  TurbolinksAnimate.init({element: document.querySelector('body.turbolinks-animate'), animation: 'fadeinup'})
```

Comment:
This is going to call init on every single page load, is this intended behavior?

Reply:
Oops, not sure but this was leftover code from a library I ended up not using. Removed!